### PR TITLE
open cart doc improvement

### PR DESCRIPTION
**Pull request by TChukwuleta**
Part implementation of #1352 for Open cart.

### CODE REVIEW

**Comment by pavlenex** [comment]
I wont' be prioritizng a review on this one for a few weeks, quick glance it's really overwhelmingly screams chatgpt. If you could actually proofread it yourself and rephrase I could then prioritize it.

File: docs/OpenCart.md [hunk 1]
```diff
@@ -6,13 +6,51 @@ tags:
   - eCommerce
 ---
 
-# OpenCart integration
+# BTCPay Server Integration with OpenCart: A Complete Guide
```
Comment (pavlenex) [inline docs/OpenCart.md:9]:
```suggestion
# Accept Bitcoin payments in OpenCart
```

File: docs/OpenCart.md [hunk 2]
```diff
@@ -6,13 +6,51 @@ tags:
...
+
+[BTCPay Server](https://btcpayserver.org/) is a free and open-source cryptocurrency payment processor. It allows you receive payments in Bitcoin and altcoins directly into your wallet with no fees, transaction cost or a middleman.
```
Comment (pavlenex) [inline docs/OpenCart.md:11]: Please, if you're doing chatgpt proofread the output. How is there no transaction cost?
```suggestion
[BTCPay Server](https://btcpayserver.org/) is a free and open-source bitcoin payment processor. It allows you receive payments directly into your wallet with no fees or a middleman.
```

File: docs/OpenCart.md [hunk 3]
```diff
@@ -6,13 +6,51 @@ tags:
...
+
+ If you run an OpenCart store, integrate your store today with BTCPay Server to help expand your payment options, attract crypto-savvy customers, without paying a dime.
```
Comment (pavlenex) [inline docs/OpenCart.md:13]: Unnecessary marketing buzzwords
```suggestion
 If you run an OpenCart store, integrate your store today with BTCPay Server to help expand your payment options, attract crypto-savvy customers, without paying a dime.
```

File: docs/OpenCart.md [hunk 4]
```diff
@@ -6,13 +6,51 @@ tags:
...
+ 
+This guide covers the integration process for both OpenCart 3 and OpenCart 4, helping you set up BTCPay Server seamlessly.
+
+
+## Why Choose BTCPay Server: Unmatched Benefits for Your Payment Processing
+
+BTCPay Server is a non-custodial invoicing system that ensures complete control over your payments by eliminating third-party involvement. With the BTCPay Server OpenCart extension, payments are sent directly to your wallet, ensuring both privacy and security.
```
Comment (pavlenex) [inline docs/OpenCart.md:20]: Didn't we already mention this on line 11, why repeating it?
```suggestion
```

File: docs/OpenCart.md [hunk 5]
```diff
@@ -6,13 +6,51 @@ tags:
...
+# BTCPay Server Integration with OpenCart: A Complete Guide
+
+[BTCPay Server](https://btcpayserver.org/) is a free and open-source cryptocurrency payment processor. It allows you receive payments in Bitcoin and altcoins directly into your wallet with no fees, transaction cost or a middleman.
+
+ If you run an OpenCart store, integrate your store today with BTCPay Server to help expand your payment options, attract crypto-savvy customers, without paying a dime.
+ 
+This guide covers the integration process for both OpenCart 3 and OpenCart 4, helping you set up BTCPay Server seamlessly.
+
+
+## Why Choose BTCPay Server: Unmatched Benefits for Your Payment Processing
```
Comment (pavlenex) [inline docs/OpenCart.md:18]:
```suggestion
## Benefits of using BTCPay for OpenCart
```

File: docs/OpenCart.md [hunk 6]
```diff
@@ -6,13 +6,51 @@ tags:
...
+
+BTCPay Server is a non-custodial invoicing system that ensures complete control over your payments by eliminating third-party involvement. With the BTCPay Server OpenCart extension, payments are sent directly to your wallet, ensuring both privacy and security.
+
+BTCPay Server can be deployed as a self-hosted solution on your server, or you can opt for a third-party host. The self-hosted option not only allows you to manage an unlimited number of stores and leverage the Lightning Network, it also gives you the flexibility to act as a payment processor for others.
```
Comment (pavlenex) [inline docs/OpenCart.md:22]: Be concise here, let's just use bulletpoints, this chatgpt mumbo jumbo is becoming annoying and adds zero value just words.

**Review by pavlenex — CHANGES_REQUESTED** [review]
@TChukwuleta As said above, if you're using chatgpt, please proofread to ensure accuracy before pushing a PR, it takes more time to edit out this PR from repetative unecessary marketing buzzwords than it takes to generate it.

Most of the changes are okay, but please re-write the feature and benefits section according to my comments.

**Comment by TChukwuleta** [comment]
Reviewed them... Apologies for the stress...

Can you take another look @pavlenex

**Review by pavlenex — APPROVED** [review]

**Pull request merged**